packages/snaps-utils/src/test-utils/snap.ts

```diff
@@ -27,6 +28,16 @@ export const MOCK_SNAP_ID = 'npm:@metamask/example-snap';
 export const MOCK_LOCAL_SNAP_ID = 'local:@metamask/example-snap';
 export const MOCK_ORIGIN = 'example.com';
 
+type GetPersistedSnapObjectOptions = Partial<Omit<PersistedSnap, 'version'>> & {
+  version?: string | SemVerVersion;
```

Comment:
Why do we accept `string` here?

Reply:
https://github.com/MetaMask/snaps-monorepo/pull/961#discussion_r1024199047